Update instruction.rs

sladar1 opened the pull request:
Adding opcodes 0xb0 - 0xff

sladar1 committed:
Update instruction.rs

Adding opcodes 0xb0 - 0xff

toddradin commented on src/instruction.rs:
```diff
@@ -511,34 +511,410 @@ impl Instruction {
                 cycles: 5
             },
 
```
Nit: you can remove this blank line

toddradin commented on src/instruction.rs:
```diff
@@ -511,34 +511,410 @@ impl Instruction {
-
+            //0xb0 through 0xff - DS
+            0xb0 => Instruction {
+                op: Operation::ORA(Register::B),
```
You will need to add all of the new operations you have added here to the Operation enum. This should not compile otherwise.

sladar1 committed:
0xb0 - 0xff, space removed, commands added

Went through and added the commands to the Operation enum. I believe I got them all, though let me know if I missed any.

Commit:
0xb0 - 0xff added debug info

Compiles on my end. Teddie's section isn't present here so it doesn't run for very long before stopping due to the unimplemented instructions.

arnolth committed:
Merge branch 'master' into opcodes-0xB0-FF

arnolth committed:
Update instruction.rs
